system/lib/pthread/pthread_create.c

```diff
@@ -171,10 +204,13 @@ int __pthread_create(pthread_t* restrict res,
   new->prev->next = new;
 
   __tl_unlock();
+
+#ifdef PTHREAD_DEBUG
```

Comment:
if this is behind the ifdef, should it just be left uncommented?

Reply:
Done